packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -35,7 +35,8 @@ contract TaikoL1 is TaikoL1Base {
             proofRegularCooldown: 30 minutes,
             proofOracleCooldown: 15 minutes,
             proofWindow: 90 minutes,
-            proverBond: 10_240e18,
+            proverBondOp: 10_240e18,
```

Comment:
There is no ZK or OP in the core protocol codebase please. All proofs have a tier, each tier has its own configuration.